cardano-api/src/Cardano/Api/Query.hs

```diff
@@ -290,7 +304,10 @@ fromShelleyRewardAccounts =
 toConsensusQuery :: forall mode block result.
                     ConsensusBlockForMode mode ~ block
                  => QueryInMode mode result
-                 -> Some (Consensus.Query block)
+                 -> Some (Ledger.Query block)
+toConsensusQuery (QuerySystemStart CardanoModeIsMultiEra) =
+    Some Ledger.GetSystemStart
```

Comment:
The notion of queries is a consensus notion, not ledger. And it uses the consensus API, not ledger API.

Reply:
Fixed!